lib/gollum/views/helpers/locale_helpers.rb

```diff
@@ -0,0 +1,75 @@
+module Precious
+  module Views
+    module LocaleHelpers
+      NO_METHOD_MESSAGE = "Argument must be a view method"
+
+      # Returns all I18n translation strings for the current view class.
+      # This method support YAML arguments. For example:
+      #
+      #     last_edited: This content was last edited at %{date}.
+      #
+      # Where the `date` argument must be a method available on the current
+      # class.
+      #
+      # Use this interface within Mustache templates to render any user
+      # interface strings in the current locale. For example:
+      #
+      #     {{ t.last_edited }}
+      #
+      def t
+        autofill I18n.t(locale_klass_name)
+      end
+
+      private
+
+      # Recursively looks up I18n translation values and autofills any YAML
+      # arguments with the return value of the current class's matching method.
+      #
+      # When a translation value with an argument has no matching method, we
+      # then return that value transformed to include the `no_method_message`
+      #
+      def autofill(yaml)
+          yaml.map { |i18n_key, i18n_value|
+          if i18n_value.is_a? Hash
+            [i18n_key, autofill(i18n_value)]
+          else
+            yaml_variable_regex = /\%\{[\w]+\}/
+            yaml_arguments = i18n_value.scan(yaml_variable_regex).uniq
+
+            next [i18n_key, i18n_value] if yaml_arguments.empty?
+
+            yaml_arguments.each do |argument|
+              non_word_character_regex = /[^\w]/
+              method_name = argument.gsub(non_word_character_regex, "")
```

Comment:
`''` is more efficient than `""` (at least in older versions of ruby -- maybe this has changed recently?)

Reply:
I will have to do more thorough research, but my understanding is that the performance loss is negligible in recent Rubies. 

But really you are right, I have just done this more as a habit, as some projects I work on require double-quotes for all strings.